src/components/storagePools/storageVolumeCreateBody.tsx

```diff
@@ -33,19 +33,19 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
-export interface ValidationFailed {
-    volumeName?: string | undefined;
-    size?: string | undefined;
-}
-
 export interface DialogValues {
     volumeName: string,
     size: number,
     unit: string;
     format: optString;
 }
 
-type OnValueChanged = <K extends keyof DialogValues>(key: K, value: string) => void;
+export interface ValidationFailed {
+    volumeName?: string | undefined;
+    size?: string | undefined;
```

Comment:
Is it really necessary to have both `?` and `| undefined`? That looks redundant and confusing. Depending on what the intent is (size is always present or only set in some cases) it should be one or the other.